packages/p2p-v2/src/pages/my-ads/components/AdFormTextArea/AdFormTextArea.scss

```diff
@@ -0,0 +1,9 @@
+.p2p-v2-ad-form-textarea {
+    & .deriv-textarea {
+        &__footer {
+            & .deriv-text {
+                font-size: 1.2rem;
```

Comment:
can't you pass textSize to the TextArea component? i believe size sm is 1.2rem

Reply:
hmmm cause i checked in the storybook and the textSize is applied to both the texts 👀

Reply:
yes, but i need 14px for text size and 12px for hint size.

Reply:
dang 😆 maybe you can suggest to deriv-com team to pass two different text size props 🙏 but can resolve this comment